src/ui/common/src/components/workflows/edges/AqueductBezier.tsx

```diff
@@ -28,7 +28,6 @@ export const AqueductBezier: React.FC<EdgeProps> = ({
       <defs>
         <marker
           id="arrow-closed"
-          viewBox="-10 -10 20 20"
```

Comment:
Linting gave me the following error so I removed the viewBox:
`Invalid property 'viewBox' found on tag 'marker', but it is only allowed on: svg  `

Reply:
This is strange that this error is just now showing up. 

Could you please file a Linear task for this and reply here so that we can investigate this issue later?

Reply:
Same as my other comments on this viewBox issue. Could you please comment out this line instead of removing and add a TODO: comment to investigate further?

A linear task link in the TODO comment would be awesome.

Reply:
Here is the linear task: https://linear.app/aqueducthq/issue/ENG-1739/look-into-linter-complaint-invalid-property-viewbox